Example/ViewController.swift

```diff
@@ -241,35 +280,52 @@ class ViewController: UIViewController {
         guard let route = routes?.first else { return }
 
         let styles = [CustomDayStyle(), CustomNightStyle()]
-        let options = NavigationOptions(styles:styles, navigationService: navigationService())
+        let options = NavigationOptions(styles:styles, navigationService: navigationService(route: route))
         let navigationViewController = NavigationViewController(for: route, options: options)
         navigationViewController.delegate = self
 
-        presentAndRemoveMapview(navigationViewController)
+        presentAndRemoveMapview(navigationViewController, completion: beginCarPlayNavigation)
     }
 
-    func navigationService() -> NavigationService? {
-        guard let route = routes?.first else { return nil }
+    func navigationService(route: Route) -> NavigationService {
         let simulate = simulationButton.isSelected
-        let mode: SimulationMode = simulate ? .always : .never//.onPoorGPS
+        let mode: SimulationMode = simulate ? .always : .onPoorGPS
```

Comment:
This fixes an issue where the example application didn’t automatically simulate movement within tunnels.